src/deepsparse/utils/state.py

```diff
@@ -62,3 +92,24 @@ def update_value(self, attribute: str, value: Union[str, int, list]):
 
     def update_state(self, value: Any):
         self._current_state.update(value)
+
+    def get_state(self, key):
+        if key in self.current_state:
```

Comment:
docstring?

Reply:
Doc strings are inconsistent in this repo. Here above, update_state dont even have any.
Ill address this in the stand up